package.json

```diff
@@ -4,6 +4,9 @@
   "author": "Takuya Asano <takuya.a@gmail.com>",
   "description": "JavaScript implementation of Japanese morphological analyzer",
   "main": "./dist/node/kuromoji.js",
+  "browser": {
+    "./dist/node/loader/NodeDictionaryLoader.js": "./dist/node/loader/BrowserDictionaryLoader.js"
```

Comment:
Hmm... `gulp build` does not seem to need `b.require` in my environment.

my package.json:
```
  "browser": {
    "./src/loader/NodeDictionaryLoader.js": "./src/loader/BrowserDictionaryLoader.js"
  },
```
my gulpfile.js:
```
    var b = browserify({
        entries: ["./src/kuromoji.js"],
        standalone: "kuromoji" // window.kuromoji
    });
    // Commented out b.require
    // b.require(__dirname + "/src/loader/BrowserDictionaryLoader.js", {expose: "./loader/NodeDictionaryLoader.js"});
    b.bundle()
        .pipe(source("kuromoji.js"))
        .pipe(gulp.dest("./dist/browser/"));
```
I executed
```
$ gulp build
```
then I got preferred `dist/browser/kuromoji.js` with BrowserDictionaryLoader.